src/plugins/content/accessibility/keyNavigation.js

```diff
@@ -523,6 +523,30 @@ function initDefaultItemNavigation(testRunner) {
     return itemNavigators;
 }
 
+function initTTSNavigation(testRunner) {
+    const pluginName = 'apiptts';
+    const actionPrefix = `tool-${pluginName}-`;
```

Comment:
Please avoid introducing coupling like that. APIP TTS is a plugin, and therefore should not be directly expected by a generic component.

Eventually, you may add another generic group. But not hard coupled.

As an idea, you might create a group based on a particular css class. And use this class to tag the plugin elements. This is how this is made for any other groups.